de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/core/ui/util/CollaborationUtils.java

```diff
@@ -301,8 +308,9 @@ private static String getShareProjectDescription(
      * @param sarosSession
      * @return
      */
-    private static Map<IProject, List<IResource>> acquireResources(
-        List<IResource> selectedResources, ISarosSession sarosSession) {
+    private static Map<IReferencePoint, List<IResource>> acquireResources(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this method to reduce its Cognitive Complexity from 28 to the 15 allowed. ([squid:S3776](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3776))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWaWrffYsSPV6ZXI_B9b&open=AWaWrffYsSPV6ZXI_B9b)

[ISSUE_KEY]: AWaWrffYsSPV6ZXI_B9b